Projects/UOContent/Multis/Houses/BaseHouse.cs

```diff
@@ -2070,7 +2070,7 @@ private void SetLockdown(Item i, bool locked, bool checkContains = false)
 
         public bool LockDown(Mobile m, Item item, bool checkIsInside)
         {
-            if (!IsCoOwner(m) || !IsActive)
+            if (!IsFriend(m) || !IsActive)
```

Comment:
I am not sure if this is right. Let's do some testing.

Reply:
Friends of the house have always been allowed to lock things down. This was tested via OSI and is at the link: https://uo.com/wiki/ultima-online-wiki/gameplay/houses-placing-a-house/house-ownership-4-managing-your-home/